### ISSUE TITLE
Returns NULL in Android 10

### ISSUE THREAD

**Issue by c7nasr** [pocesar/react-native-sim-data#20]
Device: Honor 10 Lite
SIM: 2 sim card
Permission: Granted 
Result: Returns Null

**Comment by pocesar** [comment]
does it work after restarting?

**Comment by c7nasr** [comment]
nope. not working even after clean the cache and reinstall Node Modules.

**Comment by xhbisme** [comment]
I met the same problem，phoneNumber0 is null, but phoneNumber1 is ok,

**Comment by orhangazi** [comment]
It's return null with expo. Can i get sim info using expo?

**Comment by Chasty** [comment]
any updates I'm receving null too

**Comment by zishon89us** [comment]
Using android 10, getting `null` on both `RNSimData.getSimInfo().phoneNumber1` and `RNSimData.getSimInfo().phoneNumber0`, error looks like this:

> TypeError: null is not an object (evaluating '_reactNativeSimData.default.getSimInfo().phoneNumber1')

any suggestions or hints how to make this work?
